### DISCUSSION TITLE
child theme and template parts

### DISCUSSION THREAD

**Discussion by PiNkjeLLyFaLL** [elementor/elementor#14125] [Developers Community]
Hello all,
I have a question that I cannot find an answer, I apologize and thank you for your indulgence if this has already been resolved 🙏

I am using **ELEMENTOR PRO** with theme builder to create header, footer, archive... and other elements
I did this with the **Hello Elementor _theme_**.

Now that my site's development is progressing, I wanted to create a **child theme** for my custom CSS and other functions.
but I realize that all custom parts created with the theme builder are not related to my child theme and only appear on Hello Elementor theme

do I have to start all over from 0? or is it possible to move the theme builder items to a child theme?

Thank you for your answers !!

Elementor Version Version 3.1.1
Elementor pro version 3.0.10

ps: message automatically translated with google, if anyone speaks french i will be happy to read it

**Comment by PiNkjeLLyFaLL** [comment]
up! 
please somebody help me 🙏

**Comment by d19dotca** [comment]
This doesn't directly answer your question but in general child-themes are no longer needed except in very rare circumstances. If you're using custom CSS inside of Elementor as a site-wide CSS, then you can put it there instead of needing to add to any child themes. There's almost no reason to use child-themes these days if using both Elementor and Elementor Pro.
